pkg/cmd/openshift-kube-apiserver/admission/imagepolicy/imagepolicy.go

```diff
@@ -62,28 +62,31 @@ func Register(plugins *admission.Plugins) {
 				return nil, errs.ToAggregate()
 			}
 			klog.V(5).Infof("%s admission controller loaded with config: %#v", imagepolicy.PluginName, config)
-			return newImagePolicyPlugin(config)
+			return NewImagePolicyPlugin(config)
 		})
 }
 
-type imagePolicyPlugin struct {
+type ImagePolicyPlugin struct {
```

Comment:
I'm surprised by this, but I'm still reading.

Reply:
The split required me to be able to inject it inside values for testing. That was the simplest, iow. my laziness :wink: